docs/operations/metrics.md

```diff
@@ -336,9 +336,9 @@ These metrics are for the Druid Coordinator and are reset each time the Coordina
 
 ### Service Health
 
-|Metric|Description|Dimensions|Normal value|
-|------|-----------|----------|------------|
-| `service/heartbeat` | Metric indicating the service is up. `ServiceStatusMonitor` must be enabled. |`leader` on the Overlord and Coordinator.|1|
+|Metric|Description| Dimensions                                                                                                                                                                              |Normal value|
+|------|-----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------|
```

Comment:
Nit: I am not sure if this is the format we follow in the docs. The original one made more sense to me.

Reply:
This is what intelliJ did when I edited the other lines in the table. I can undo these changes, but are there some settings I can set in intelliJ to auto lint the markdown to match the expected styleguide?